### DISCUSSION TITLE
Where do `console.log` statements in `editorStatus.ts` show up?

### DISCUSSION THREAD

**Discussion by hirehamir** [microsoft/vscode-discussions#1244] [Extension Development QnA] [answered]
Where do `console.log` statements in `src/vs/workbench/browser/parts/editor/editorStatus.ts` show up?

**Comment by isidorn** [comment, marked as answer]
There are no console.log statements in that file.
But if you are changing that file and adding statements they will show up in the Browser Developer Tools.
